src/Components/Pages/ContributorsPage.js

```diff
@@ -5,11 +5,11 @@ import { _ } from 'Utils';
 const Card = props => {
   const { title, name, github, about, website } = props;
   return (
-    <div className="card">
-      <div className="card-name">{name}</div>
-      <div className="card-image">
-        <img src="https://i.imgur.com/IyxFgGe.png" alt=""></img>
+    <div className="card border-0">
+      <div className="card-image my-2">
+        <img src="https://i.imgur.com/IyxFgGe.png" alt="" className="w-25 rounded-circle"></img>
       </div>
+      <div className="card-name"><h4>{name}</h4></div>
       <div className="card-title">{title}</div>
       <div className="card-github">{github}</div>
```

Comment:
github link should probably be an <a> tag that opens a new tab

Reply:
website too!